### PR TITLE
Move patterns repo to da handbook

### PR DESCRIPTION
**Pull request by pablorc**
fusion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ For now, we have the following resources available:
 
 1. [Our project management guidelines](/guides/pm-guidelines.md)
 1. [Our Git & Commit guidelines](/guides/git-guidelines.md)
-1. [Some of our used programming patterns](https://github.com/MarsBased/patterns)
+1. [Some of our used programming patterns](/guides/patternd/index.md)
```
Comment (artero) [inline README.md:31]:
```suggestion
1. [Some of our used programming patterns](/guides/patterns/index.md)
```

File: guides/patterns/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# MarsBased Patterns
+
+A collection of MarsBased programming patterns in different languages.
+
+## Ruby on Rails
+
+- [Presenter](rails/presenter.md)
+- [Command](rails/command.md)
```
Comment (artero) [inline guides/patterns/index.md:8]: Can we add the Pattern Query Object too, please 🙏 (I love It!)

https://github.com/MarsBased/patterns/commit/f771d64c5412d11c4d6a372c8168c89ed3391ba8

Comment (pablorc) [reply guides/patterns/index.md:8]: I only need a re-review there before adding it in this repo. I moved the structure from multiple files to a single Markdown and I'd like new approvals.

**Comment by alexrodba** [comment]
NOICE

**Pull request merged**